### ISSUE TITLE
Updating to ColorControl 8.1.0.0 issue

### ISSUE THREAD

**Issue by javsp** [Maassoft/ColorControl#139]
When I update the app I usually end the ColorControl process and then I overwrite the folder. This time I couldn't do that because it said some files were being used yet. So I opened services.msc and manually stopped the ColorControl service. I could overwrite the files and then I restarted the service again.

Apparently it's working nice but Is this the correct way to update this app?

**Comment by Maassoft** [comment]
Yes, but I think this is not very user-friendly. Maybe I'll build in an auto-updater later.
